generators/server/index.js

```diff
@@ -441,6 +441,26 @@ module.exports = JhipsterServerGenerator.extend({
                 },
                 {
                     type: 'list',
+                    name: 'container',
```

Comment:
I would rather have this as a CLI option else we need to mark jetty and undertow as beta